Rewrite linux installation guide

hentrope opened the pull request:
Followup to [PR 215](https://github.com/SlimeVR/SlimeVR-Docs-Site/pull/215).
- I updated instructions to use the AppImage from the latest release (much simpler to use)
- I added a small section on firewall rules in Linux
- I moved all previous instructions to a "Legacy Setup" section for now. Should this be removed entirely?

ImUrX commented:
I haven't read this but can you also add a part on the serial permissions? You can't do the wifi setup otherwise

hentrope commented:
> I haven't read this but can you also add a part on the serial permissions? You can't do the wifi setup otherwise

I don't have actual slimes yet, I've just been tinkering with this with owoTrack for now. Not really familiar with what needs to be done for serial.

hentrope commented:
I've added an additional section with what I believe should be the recommended way to set up the permissions for Serial Console. I've seen alternatives suggested on Discord though, so I'm open to changing the recommended method.

I don't have Slimes yet, but I do have the D1 Minis. I was able to get the console to print out a bunch of gibberish after setting up the udev rules, so I _think_ it works.

ImUrX commented:
eh, its a universal solution so good enough imo

calliePepper commented:
Thank you so much for writing this up, we really appreciate it!

hentrope commented:
Pushed up one more change. I couldn't find any instructions specific to installing the OpenVR plugin on Linux, so I added them here.

ImUrX commented:
do you mind adding the java 17 requirement, you can put it next to the appimage like what the old guide suggests or you can just install it with your package manager. SlimeVR will find it :p

hentrope commented:
> do you mind adding the java 17 requirement, you can put it next to the appimage like what the old guide suggests or you can just install it with your package manager. SlimeVR will find it :p

Sure, I can do that. Didn't realize it was a requirement because it just worked on my machine, I assumed it was bundled in the AppImage

ImUrX commented:
It's not, I should but it would increase binary size so much :sob:

hentrope commented:
Reworked the Java section, tried my best to make it apply to both the normal install as well as the legacy install (this might be a bit clunky, let me know if anything should be reworded). Also added a tidbit on installing Java globally.
Plenty of stuff for you guys to review :wink:
